robot.py

```diff
@@ -22,10 +26,14 @@ def teleopInit(self):
         self.running = 0
```

Comment:
This is shooter state, so should be self.shooter.stop()  or a method you prefer.
If you are using running as a speed, then this should be speed, otherwise use a boolean variable

in Shooter.stop(self):
       self.is_running = False

Reply:
It would feel weird having a self.shooter.stop() in the init, especially because having the two shooter variables be initialized there is completely unnecessary. Should I just delete those two initializations? Having a Shooter.stop() seems like a good idea though.

Reply:
My comment was mostly a general statement about where the shooter state should be.  (inside the shooter object, not inside the robot object).       That should make things clean and give you the ideal place to initialize things at the appropriate time.